extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -160,6 +161,23 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Prefetches and preconnects URLs related to the analytics.
+   * @param {boolean=} opt_onLayout
+   * @override
+   */
+  preconnectCallback(opt_onLayout) {
+    let url;
+    if (getMode().localDev || getMode().test) {
+      const loc = this.getAmpDoc().win.parent.location;
```

Comment:
Either add the following or put in the if clause:
dev().assert(this.getAmpDoc().win.parent && this.getAmpDoc().win.parent.location);

Reply:
done.